package me.ninjoh.nincore.api.exceptions.technicalexceptions;


import me.ninjoh.nincore.api.exceptions.TechnicalException;

public class SubCommandAlreadyExistsException extends TechnicalException
{

}